README.md

```diff
@@ -140,7 +140,7 @@ See also [Static Site Generators](#static-site-generators), [Content Management
 
 Some [Groupware](#groupware) solutions also feature calendar/address book editing and synchronization.
 
-  * [AgenDAV](http://agendav.org/) - Multilanguage CalDAV web client with a rich AJAX interface and shared calendars support
+  * [AgenDAV](http://agendav.org/) - Multilanguage CalDAV web client with a rich AJAX interface and shared calendars support [Source Code](https://github.com/adobo/agendav) `GPLv3` `JavaScript` `PHP`
```

Comment:
PHP only. JS is only used client side in the browser. PHP is the required platform on the server.